README.md

```diff
@@ -27,31 +22,20 @@ sudo sh /scripts/dev_deps.sh
 To run the Entitlements API locally, you will need an Enterprise Services cert with access to the dev subscription endpoint /search/criteria.
 
 * You can request a personal cert by following ALL steps in this [mojo doc](https://mojo.redhat.com/docs/DOC-1144091).
-* You should be emailed a link that will allow you to import your pk12 cert into Firefox.
-
-After importing the pk12 cert into Firefox, you can export it into a separate .crt and .key file that can be used to
-query subscription services. To export your .crt and .key file:
-
-* Export your pk12 cert to your local box:
-  * Go to Firefox preferences
-  * Select Privacy & Security
-  * Select View Certificates
-  * Select your pk12 cert
-  * Select Backup...
-  * Save as a pk12 file  
-* From here you can export your crt and key like so:
+* You can export your crt and key like so:
     `openssl pkcs12 -in your-p12-cert.p12 -out your-key.key -nocerts -nodes`
     `openssl pkcs12 -in your-p12-cert.p12 -out your-cert-sans-key.crt -clcerts -nokeys`
 
 ## Create your config file
 
-You'll need to make a config file specific to your machine. Create a file `/config/development.{your_user_name}.sh` and enter the following:
+You'll need to make a config file specific to your machine.
+Create a file `mkdir -p ./local && ./local/development.env.sh` and enter the following:
```

Comment:
You sure this syntax is right? I feel like it should be `mkdir -p ./local && touch ./local/development.env.sh` or something like that

Reply:
Yeah I like it... gonna change